content/machine-learning/regreg.Rmarkdown

```diff
@@ -0,0 +1,129 @@
+---
+title: Regularized Regression Classifier
+weight: 15
+draft: false
+---
+
+{{% author %}}By Frederik Hjorth{{% /author %}} 
+
+
+Regularized regression is a classification technique where the category of interest is regressed on text features using a penalized form of regression where parameter estimates are biased towards zero. Here we will be using a specific type of regularized regression, the Least Absolute Shrinkage and Selection Operator (or simply 'lasso'), but its main alternative, ridge regression, is conceptually very similar. 
+
+In the lasso estimator, the degree of penalization is determined by the regularization parameter $\lambda$: 
+
+$$
```

Comment:
Makes sense - I'll drop the explanation part. I will need to mention the lambda parameter though. And sorry, new to MathJax - would it accept $\lambda$ ?

Reply:
You can use the Unicode character λ or `u03BB`.

https://en.wikipedia.org/wiki/Lambda#Character_encodings

Reply:
I built the page locally. It is probably better to say just `lambda` (like `alpha`) because there is no equation anymore. Also `lambda_best` is better than `best.lambda` because we don't use dots.